src/lib/components/workflow/workflows-summary-configurable-table/filterable-table-cell.svelte

```diff
@@ -46,11 +57,23 @@
       $workflowFilters = [...getOtherFilters()];
     }
 
-    updateQueryParamsFromFilter($page.url, $workflowFilters);
+    updateQueryParamsFromFilter(page.url, $workflowFilters);
   };
 </script>
 
-{#if href}
+{#if isRunId}
+  <Tooltip text={value} top class="min-w-0">
+    {#if href}
```

Comment:
I don't think we need this if statement. RunIds will always have href

Reply:
Okay awesome! Thanks Tideman